lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -365,6 +375,15 @@ private Map<String, Object> buildReuseResponse(AuditEventUser auditEventUser)
                         auditEventUser));
 
         LOGGER.info(LogHelper.buildLogMessage("Returning reuse journey"));
+
+        if (attainedVot.getProfileType() == OPERATIONAL_HMRC) {
+            // the only VC we should possibly have collected this session at this point is a
+            // migration VC
+            return vcReceivedThisSession.isEmpty()
```

Comment:
Is this too lazy a check? At this stage in the journey the users should have only ever picked up an inherited identity VC, so this works. But should we be doing further scrutiny on it, to make sure it actually IS an inherited identity VC?